service-workers/cache-storage/resources/cache-keys-attributes-for-service-worker.js

```diff
@@ -0,0 +1,23 @@
+self.addEventListener('fetch', function(event) {
+    const url = event.request.url;
+    if (url.endsWith('?ignore')) {
+      return;
+    }
+    const match = url.match(/\?name=(\w*)/)
```

Comment:
Can we use the URL type to simplify parsing? e.g.

const params = new URL(event.request.url).searchParams;
if (params.has('ignore')) ...

if (!params.has('name')) ...

const name = params.get('name');

Reply:
Done.